eth/v1alpha1/node.proto

```diff
@@ -125,3 +144,11 @@ enum PeerDirection {
   INBOUND = 1;
   OUTBOUND = 2;
 }
+
+// ConnectionState states the current status of the peer.
+enum ConnectionState {
```

Comment:
Given `0` is the default, should we do `UNKNOWN = 0` ?

Similar to:
```
enum ValidatorStatus {
  UNKNOWN_STATUS = 0;
  ...
}
```

Reply:
The default would be `Disconnected` for any peer. Maybe I can make that the default then ?

Reply:
I think `Disconnected` as 0 makes the most sense